docs/guides/embedding-content.rst

```diff
@@ -73,6 +73,13 @@ from our own docs and will populate the content of it into the ``#help-container
 You can modify this example to subscribe to ``.onclick`` Javascript event,
 and show a modal when the user clicks in a "Help" link.
 
+.. tip::
+
+   You can use a `Sphinx reference`_ as ``section`` argument instead of the slug of the title (i.e. ``creating-an-automation-rule``)
```

Comment:
I think an example is better here, this `i.e.` section doesn't make sense to me?

Reply:
Yeah. I don't know what I was thinking when I wrote this. I'm reading it now and I don't understand it :stuck_out_tongue_closed_eyes: